source/tutorials/nix-language.md

```diff
@@ -990,7 +990,7 @@ Wherever you see a colon (`:`) in Nix language code:
 - On its right is the function body.
 
 Function declarations in the Nix language can appear in different forms.
-Each of them is explained in the following, and here is an overview:
+Each of them is explained below, and here is an overview:
```

Comment:
I disagree with saying "below" or "above", because on the web the publisher has no final control over presentation. Anyone is free to display this in multiple columns, pages, backwards, whatever.